ticdc: update some usage restrictions and configs (#3881)
pingcap/docs: ticdc/troubleshoot-ticdc.md

> **Note:**
>
> + TiCDC (the 4.0 version) does not support dynamically modifying the file sorting and memory sorting yet.

> > + Currently, the file sorting feature only has limited processing capacity. If too large a  data volume on a single table causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.

Reviewer: ```suggestion
> + Currently, the file sorting feature only has limited processing capacity. If accumulating data of a single table is too large and causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.
```

Author: What about the following version?
```suggestion
> + Currently, the file sorting feature only has limited processing capacity. If the data accumulated on a single table is too large and causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.
```

Reviewer: looks good

> > + Currently, the file sorting feature only has limited processing capacity. If too large a  data volume on a single table causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.

Reviewer 2: ```suggestion
> + Currently, the file sorting feature only has limited processing capacity. If too large a data volume on a single table causes the file sorting to fail, you can modify the task configuration of TiCDC to filter out this table and use other backup and restore tools (such as [BR](/br/backup-and-restore-tool.md)) to restore the table before you resume replicating the table.
```

Author: Applied in https://github.com/pingcap/docs/pull/3881#discussion_r486778489.
